services/lsst-login.rst

```diff
@@ -89,9 +89,93 @@ When using an ``lsst-login`` node as a "jump host" you may also wish to configur
       ProxyJump lsst-login01.ncsa.illinois.edu
       DynamicForward yourportnumber
 
-You may also wish to reuse a single connection to/through an ``lsst-login`` node via a control socket/multiplexing. See for example
+You may also wish to reuse a single connection to/through an ``lsst-login`` node via a control socket/multiplexing. This means you can authenticate to the login node once, and reuse that connection in a 2nd terminal without authenticating again. See for example
 `OpenSSH Cookbook - Multiplexing <https://en.wikibooks.org/wiki/OpenSSH/Cookbook/Multiplexing>`_.
 
+A relatively complete ``~/.ssh/config`` "recipe" for streamlining your SSH connections (assuming OpenSSH, e.g., on Linux or macOS) through the ``lsst-login`` nodes might look like this:
+
+.. prompt:: bash $ auto
+
+   # Set common config for the lsst-login nodes
+   Host lsst-login*
+      # if your account on your local workstation/laptop does not match your LSST username, indicate the latter should be used;
+      # substitute your own NCSA username
+      User ncsausername               
+      # allow use of a Kerberos ticket on your local machine for auth to LSST machines
+      GSSAPIAuthentication yes   
+      # prefer Kerberos ticket auth, amongst other possibilities (order/include others as desired)
+      PreferredAuthentications gssapi-with-mic,keyboard-interactive,password
+      # forward your local Kerberos ticket to the login node if you need to continue to another LSST server after the login
+      GSSAPIDelegateCredentials yes
+      # configure OpenSSH Control Master "multiplexing" (to allow reuse of an initial connection)
+      ControlMaster auto
+      ControlPath ~/.ssh/cm_socket_%r@%h:%p
+      ControlPersist 24h
```

Comment:
I'm not sure we really want to suggest `ControlPersist 24h`. That allows them to reconnect up to 24 hours after they last disconnected. I generally set mine to 30 seconds. I'd definitely suggest something  less than `5m.`

Reply:
Thanks! I misunderstood what that value was.